doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1415,7 +1417,19 @@ We are absolutely aware that it's confusing to have both ``dataverse.fqdn`` and
 dataverse.files.directory
 +++++++++++++++++++++++++
 
-This is how you configure the path Dataverse uses for temporary files. (File store specific dataverse.files.\<id\>.directory options set the permanent data storage locations.)
+Please provide an absolute path to a directory backed by some mounted file system. This directory is used for a number
+of purposes:
+
+1. ``<dataverse.files.directory>/temp`` after uploading, data is temporarily stored here for ingest and/or before
+   shipping to the final storage destination.
+2. ``<dataverse.files.directory>/sword`` a place to store uploads via the :doc:`../api/sword` before transfer
+   to final storage location and/or ingest.
+3. ``<dataverse.files.directory>/<PID Authority>/<PID Identifier>`` data location for file system imports, see
+   :ref:`api-import-dataset`.
+4. ``<dataverse.files.directory>/googlecloudkey.json`` used with :ref:`Google Cloud Configuration` for BagIt exports.
```

Comment:
You're changing this in another PR right? Is this line dropped there? Or should it just get dropped here? (FWIW - not sure that anyone but QDR is using the Google archiving so we may not need to support the legacy option of looking in this dir.)

Reply:
Yeah, let's leave it last in the list.